docs/02_cleos/03_command-reference/net/status.md

```diff
@@ -1,31 +1,66 @@
+## Command
+```sh
+cleos net status [OPTIONS] host
+```
+
+**Where:**
+* [OPTIONS] = See **Options** in the [**Command Usage**](command-usage) section below.
+* host = The hostname:port to query status of connection.
+
+**Note:** The arguments and options enclosed in square brackets are optional.
+
 ## Description
-status of existing connection
+Returns the status of a connected peer. This command allows a node operator to check the status of a node's connected peer.
 
-**Command**
+## Command Usage
+The following information shows the different positionals and options you can use with the `cleos net status` command:
 
-```sh
-cleos net status
-```
-**Output**
+### Positionals
+* `host` _TEXT_ REQUIRED - The hostname:port to query status of connection.
 
-```console
-Usage: cleos net status host
+### Options
+* `-h,--help` - Print this help message and exit.
```

Comment:
Full stop at end ... template indicate this list doesn't have them

Reply:
Removed.